bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -5273,10 +5326,7 @@ public Image getImageById(long id) throws TskCoreException {
 				long type = rs1.getLong("type"); //NON-NLS
 				long ssize = rs1.getLong("ssize"); //NON-NLS
 				String tzone = rs1.getString("tzone"); //NON-NLS
-				String md5 = "";
-				if (getSchemaVersion() > 2) {
-					md5 = rs1.getString("md5"); //NON-NLS
-				}
+				String md5 = rs1.getString("md5"); //NON-NLS
 				String name = rs1.getString("display_name");
```

Comment:
@rcordovano we discussed this, but I just thought I would point it out for review again

Reply:
Looks good to me.